src/internal/sdata/sql.go

```diff
@@ -78,9 +78,24 @@ func (m *SQLTupleWriter) GeneratePreparedStatement() (*pachsql.Stmt, error) {
 }
 
 func NewSQLTupleWriter(tx *pachsql.Tx, tableInfo *pachsql.TableInfo) *SQLTupleWriter {
-	insertStatement := fmt.Sprintf("INSERT INTO %s.%s (%s) VALUES ",
-		tableInfo.Schema,
-		tableInfo.Name,
-		strings.Join(tableInfo.ColumnNames(), ", "))
-	return &SQLTupleWriter{tx, tableInfo, insertStatement, []Tuple{}}
+	var s string
+	if tableInfo.Driver == "snowflake" {
+		var asClauses []string
+		for i, col := range tableInfo.Columns {
+			var asClause string
+			if col.DataType == "VARIANT" {
+				asClause = fmt.Sprintf(`to_variant(COLUMN%d) as %s`, i+1, col.Name)
```

Comment:
This can be simplified to just `to_variant(COLUMN%d)`, because I don't think the `as` here is doing anything.

We should explicitly add the column names as part of the `INSERT INTO`. For example, let  `co1` be a `VARIANT` type:
```
INSERT INTO <table_name> (col1, col2, ...) SELECT to_variant(COLUMN1), COLUMN2, ... FROM VALUES 
```